src/Groups/matrices/iso_nf_fq.jl

```diff
@@ -0,0 +1,172 @@
+# Detinko, Flannery, O'Brien "Recognizing finite matrix groups over infinite
+# fields", Section 4.2
+function isomorphic_group_over_finite_field(matrices::Vector{T}) where T <: Union{ fmpq_mat, AbstractAlgebra.MatElem{nf_elem} }
+   @assert !isempty(matrices)
+
+   # One should probably check whether all matrices are n by n (and invertible
+   # and such ...)
+
+   K = base_ring(matrices[1])
+   n = nrows(matrices[1])
+
+   Fq, matrices_Fq = good_reduction(matrices, 2)
+
+   G = MatrixGroup(n, Fq, matrices_Fq)
+   N = GAP.Globals.Order(G.X)
```

Comment:
This will give you sometimes an integer and sometimes a GAP integer object, which you can't compare to a Julia integer very well. Can't you just use the official way of getting the oder? I.e.,
```suggestion
   N = order(G)
```

Reply:
Sure! I was not aware of this difference.